src/snyk/common/configuration/configuration.ts

```diff
@@ -34,6 +35,12 @@ export type FeaturesConfiguration = {
   iacEnabled: boolean | undefined;
 };
 
+
+export interface IssueViewOptions {
+  ignoredIssues: boolean;
+  openIssues: boolean;
+};
+
```

Comment:
In TS `interface`s does not carry `;` but `type` does 🙈  I'm only raising it now since this project uses prettier and eslint and if you have these extensions installed, the IDE automatically updates them when opening the file which brings a `diff` later:

<img width="930" alt="image" src="https://github.com/snyk/vscode-extension/assets/1948377/4d0b6456-20f4-4725-b6fa-1e15413cb6fc">


If you don't use the linter plugins in your IDE, you can run them on the terminal :)

```bash
npm run lint                                                                                                         127 ↵

> snyk-vulnerability-scanner@0.0.0 lint
> npx eslint "src/**/*.ts"


~/vscode-extension/src/snyk/cli/services/cliService.ts
  2:46  warning  'unknown' overrides all other types in this union type  @typescript-eslint/no-redundant-type-constituents

~/vscode-extension/src/snyk/common/configuration/configuration.ts
   38:1  error  Delete `⏎`  prettier/prettier
   42:2  error  Delete `;`  prettier/prettier
  408:4  error  Delete `⏎`  prettier/prettier
```